README.md

```diff
@@ -301,6 +301,10 @@ spark.sql.adaptive.skewJoin.enabled true
 spark.shuffle.sort.io.plugin.class org.apache.spark.shuffle.celeborn.CelebornShuffleDataIO
 # Required Spark version >= 3.4.0, highly recommended to disable
 spark.dynamicAllocation.shuffleTracking.enabled false
+
+# Support ShuffleManager defined in user jar
+# Required Spark version >= 4.0.0, highly recommended to false for ShuffleManager in user-defined jar
+spark.executor.userClassPathFirst=false
```

Comment:
@cfmcgrady, other highly recommended options like `spark.dynamicAllocation.shuffleTracking.enabled` is also mentioned in startup guideline, which only remind the users to pay attention to these config options. What's inappropriate from a user's perspective?

Reply:
@cfmcgrady, `spark.executor.userClassPathFirst` is enabled in my company production environment and Zhihu. BTW, since this config is a more dangerous configuration option, it's better to mark this here for reminder.